nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/BaseTransformStrictOp.java

```diff
@@ -105,7 +105,7 @@ public boolean validateDataTypes(boolean experimentalMode) {
     @Override
     public List<org.nd4j.linalg.api.buffer.DataType> calculateOutputDataTypes(List<org.nd4j.linalg.api.buffer.DataType> dataTypes){
         //All strict tranform ops: FP in, FP out
-        Preconditions.checkState(dataTypes != null && dataTypes.size() == 1, "Expected exactly 1 input datatype for %s, got input %s", getClass(), dataTypes);
+        Preconditions.checkState(dataTypes != null && dataTypes.size() >= 1, "Expected exactly 1 input datatype for %s, got input %s", getClass(), dataTypes);
```

Comment:
Is this change necessary? For what op?
If it is, update the error message too, not just the condition

Reply:
Reverted.